mm2src/common/common.rs

```diff
@@ -940,3 +949,6 @@ pub fn spawn_abortable(fut: impl Future03<Output = ()> + Send + 'static) -> Abor
     spawn(abortable.then(|_| async {}));
     AbortOnDropHandle(handle)
 }
+
+#[inline(always)]
+pub fn get_utc_timestamp() -> i64 { Utc::now().timestamp() }
```

Comment:
Does it work in WASM? Could you please check?

Reply:
Yes, it works for WASM target as well.

Reply:
Ok great!